src/classes/GiftBatchGroupedTotals.cls

```diff
@@ -0,0 +1,48 @@
+public inherited sharing class GiftBatchGroupedTotals {
```

Comment:
All of the new classes need the copyright at the top and then a minimal apex dock header.